lib/docs/core/scraper.rb

```diff
@@ -34,7 +34,7 @@ def filters
     self.text_filters = FilterStack.new
 
     html_filters.push 'container', 'clean_html', 'normalize_urls', 'internal_urls', 'normalize_paths'
-    text_filters.push 'inner_html', 'clean_text', 'attribution'
+    text_filters.push 'home_url', 'inner_html', 'clean_text', 'attribution'
```

Comment:
`home_url` needs to happen after `inner_html`, which converts the document to a string (puts it before or after `attribution`).